projects/ngx-formentry/src/form-entry/value-adapters/encounter.adapter.spec.ts

```diff
@@ -22,6 +22,7 @@ import { DebugModeService } from './../services/debug-mode.service';
 import adultForm from '../../adult.json';
 import adultFormOrders from '../../mock/orders.json';
 import adultFormObs from '../../mock/obs.json';
+import {DiagnosisValueAdapter} from "./diagnosis.adapter";
```

Comment:
```suggestion
import { DiagnosisValueAdapter } from "./diagnosis.adapter";
```

Reply:
Really, though, this import should be with the other imports of value adapters...